src/main/java/org/worldcubeassociation/workbook/WorkbookValidator.java

```diff
@@ -150,9 +150,17 @@ else if (aDatabase != null && aDatabase.getCountries().findById(country) == null
             Cell dobCell = row.getCell(dobColIdx);
             Date dob = CellParser.parseDateCell(dobCell);
             String dobText = CellParser.parseText(dobCell, false);
-            if (dobText != null && !dobText.equals("") && dob == null) {
-                ValidationError validationError = new ValidationError(Severity.HIGH, "Misformatted date of birth", aMatchedSheet, rowIdx, dobColIdx);
-                aMatchedSheet.getValidationErrors().add(validationError);
+            if (dobText != null && !dobText.equals("")) {
```

Comment:
I believe more idiomatic java would be to do `dobText != null && dobText.length() > 0` instead.